fdbclient/NativeAPI.actor.cpp

```diff
@@ -529,6 +529,7 @@ struct WorkerInterfacesSpecialKeyImpl : SpecialKeyRangeBaseImpl {
 struct SingleSpecialKeyImpl : SpecialKeyRangeBaseImpl {
 	Future<Standalone<RangeResultRef>> getRange(Reference<ReadYourWritesTransaction> ryw,
 	                                            KeyRangeRef kr) const override {
+		if (!kr.contains(k)) return Standalone<RangeResultRef>();
```

Comment:
I was under the impression `kr.contains(k)` should always be true. Did something change here?

Reply:
Yeah, right. The current implementation should make this condition always be true.
But it is difficult to find if the condition is violated in the future accidentally.
Instead of this, we can also have an assertion here `ASSERT(kr.contains(k))`.
What's your opinion?

Reply:
Let's do `ASSERT(kr.contains(k))`

Reply:
Done. Thanks.